keps/sig-node/3063-dynamic-resource-allocation/README.md

```diff
@@ -1283,6 +1211,7 @@ instructions.
   a template are generated instead of deterministic), scheduler performance
   enhancements (no more backoff delays).
 - Kubernetes 1.29, 1.30: most blocking API calls moved into Pod binding goroutine
+- Kubernetes 1.31: removal of support for immediate allocation and for CRDs as claim parameters
```

Comment:
Are we staying in v1alpha1? Probably should bump the version if we are removing some fields.
Also worth giving a summary here of other renames.

Reply:
we are now at `v1alpha3`

Reply:
Updated, also in 4381.